py/bc.c

```diff
@@ -292,7 +292,7 @@ continue2:;
 //     MP_BC_MAKE_CLOSURE_DEFARGS
 //     MP_BC_RAISE_VARARGS
 // There are 4 special opcodes that have an extra byte only when
```

Comment:
By the way, the reason why I remembered a pending issue of #4341 is because I started to look why there're only these 4 special bytecodes which take cache byte. Having it for LOAD_METHOD would likely benefit the performance more than LOAD_ATTR. But yeah, a way how to support some caching for LOAD_METHOD is not obvious, and likely would require substantial refactors to method lookup code.